sdv/multi_table/base.py

```diff
@@ -818,19 +764,6 @@ def load_custom_constraint_classes(self, filepath, class_names):
         for synthesizer in self._table_synthesizers.values():
             synthesizer.load_custom_constraint_classes(filepath, class_names)
 
-    def add_custom_constraint_class(self, class_object, class_name):
```

Comment:
again, I think we should just raise a warning here and do a no-op. If you remove the function, any scripts calling it will start to crash